package/yast2-network.changes

```diff
@@ -1,3 +1,13 @@
+-------------------------------------------------------------------
+Thu Nov  4 15:02:24 UTC 2021 - Knut Anderssen <kanderssen@suse.com>
+
+- AutoYaST
+  - When the interface section contains the deprecated device
+    element and also the name element, then use the last one as
```

Comment:
"then use the device as name and the name as description", right?

Reply:
BTW, it would be nice to have profile "versions". That makes easier to discover what the profile really means.

Reply:
> "then use the device as name and the name as description", right?

The **device** is already used as the interface name but the "name" has precedence... I could rephrase it to make it more clear.